pnpm-lock.yaml

```diff
@@ -8935,7 +8941,7 @@ packages:
     engines: {node: ^8.13.0 || >=10.10.0}
     dependencies:
       '@grpc/proto-loader': 0.6.13
-      '@types/node': 18.0.6
```

Comment:
It seems that `@types/node` is downgraded everywhere it is a dependency. Can you please try to re-run `npx pnpm install -r`?

Reply:
I run the command and don't have nothing to commit.

![image](https://user-images.githubusercontent.com/12402324/219705491-79ab3fde-137f-4dc1-9af0-8ec50f32adb5.png)